src/app/modules/shared-components/components/score-ring/score-ring.component.html

```diff
@@ -32,8 +32,10 @@
       fill-opacity="0.4"
     />
```

Comment:
On same screenshot, the inner horizontal alignment is _slightly_ imperfect, for me it's particularly visible on `0` (I have it for both firefox and chrome)
![screenshot](https://user-images.githubusercontent.com/16257159/143915865-b2ca1f94-8929-4d76-b0fb-f8325d8b705a.png)

Reply:
It depends on font settings, 

![image](https://user-images.githubusercontent.com/7044736/144274924-2a23af5f-cdb5-4d76-8399-08e317b906aa.png)

![image](https://user-images.githubusercontent.com/7044736/144275049-2b8e87b4-51f6-498d-b52b-23e6c690e20f.png)

As you can see 100 has some padding on left, 86 has another. Horizontal alignment now does with flex, so it better then before. Also different browsers works with fonts with different way.